src/grammar/test-failed/case-19.tact

```diff
@@ -0,0 +1 @@
+fun function(,) {}
```

Comment:
We have one test case for the `Function_withVoid` non-terminal, we also need to cover the rest of them:
- `Function_withType`,
- `Function_abstractVoid`,
- `Function_abstractType`,
- `NativeFunction_withType`,
- and `NativeFunction_withVoid`.